Web/starter_project_vanguard/components/home/Landing_section/Buttons.tsx

```diff
@@ -0,0 +1,33 @@
+
+import Link from 'next/link'
+export const Buttons = () => {
```

Comment:
it might be better to use a syntax like React.FC (or whatever is the best wat for nextJS) to show what is what clearly